### PR TITLE
chore: Add arm64 architecture for some apps

### PR DESCRIPTION
**Pull request by aliesbelik**
Add arm64 architecture for some apps.

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/gron.json [hunk 1]
```diff
@@ -11,7 +11,25 @@
...
             "hash": "a532d5aa29395592f3725c336640f2cf9eb670c8ce25115aaee0082d5cdf5a0e"
+        },
+        "arm64": {
+            "url": "https://github.com/tomnomnom/gron/releases/download/v0.7.1/gron-windows-arm64-0.7.1.zip",
+            "hash": "9bd38a241f1afdbd3c8f952b92b7090e7a446cac5251bfed3fdf28f219c9dda8"
         }
     },
-    "bin": "gron.exe"
+    "bin": "gron.exe",
+    "checkver": "github",
```
Comment (rashil2000) [inline bucket/gron.json:21]:
```suggestion
```
This line can be removed, as gron doesn't get updated very often

Comment (rasa) [reply bucket/gron.json:21]: According to 
https://github.com/tomnomnom/gron/releases/
it was last released on Apr 13, 2022. Shouldn't we leave the checkver in?

Comment (niheaven) [reply bucket/gron.json:21]: Keep it IMO.

Comment (rashil2000) [reply bucket/gron.json:21]: Last code commit was in 2020 (the recent commit is only related to release binary)

Comment (rasa) [reply bucket/gron.json:21]: No, on Apr 13, 2022, they added `--values` option in the go code. See 
https://github.com/tomnomnom/gron/commit/badf401da553eb41b7ffde4be6a64809ed0ed846

**Pull request merged**